tools/generators/xcschemes/src/Generator/SchemeInfo.swift

```diff
@@ -15,7 +15,7 @@ struct SchemeInfo: Equatable {
         let scriptText: String
         let action: Action
         let isPreAction: Bool
-        let target: Target
+        let target: Target?
```

Comment:
I don't love this change but I'm not sure how else to represent the execution actions which have no associated build targets (i.e. `launch_path`)

Reply:
With your change it makes sense. And technically we could allow in the future execution actions without the macro expansion set, but the current API doesn't allow that.